signer/Cargo.toml

```
@@ -25,6 +25,11 @@ std = [
     "bip39/std",
     "schnorrkel/std",
     "secp256k1/std",
```

```suggestion
    "schnorrkel?/std",
    "secp256k1?/std",
```

In fact, these should also be optional only I think, though not an issue with this PR in itself!